### DISCUSSION TITLE
Surface is not valid error

### DISCUSSION THREAD

**Discussion by rb-03** [ultralytics/yolov5#10226] [Q&A]
Hello, I was trying to deploy your source code with my custom trained dataset. The trained YoloV5 PyTorch model was converted to TFlite and I ran a script to attach metadata to run the tflite model to give yolov5s-fps16-320-metadata.tflite as output. While doing so, I encountered a problem of Surface not Valid. Could you please suggest the solution for the same?

![WhatsApp Image 2022-11-20 at 4 29 06 PM](https://user-images.githubusercontent.com/68503150/202892799-39b1838d-c771-4c04-bcad-e293e94d4543.jpeg)

**Comment by glenn-jocher** [comment]
👋 hi, thanks for letting us know about this possible problem with YOLOv5 🚀. We've created a few short guidelines below to help users provide what we need in order to start investigating a possible problem.

## How to create a Minimal, Reproducible Example

When asking a question, people will be better able to provide help if you provide **code** that they can easily understand and use to **reproduce** the problem. This is referred to by community members as creating a [minimum reproducible example](https://stackoverflow.com/help/minimal-reproducible-example). Your code that reproduces the problem should be:

- ✅ **Minimal** – Use as little code as possible to produce the problem
- ✅ **Complete** – Provide all parts someone else needs to reproduce the problem
- ✅ **Reproducible** – Test the code you're about to provide to make sure it reproduces the problem

For [Ultralytics](https://ultralytics.com/) to provide assistance your code should also be:

- ✅ **Current** – Verify that your code is up-to-date with GitHub [master](https://github.com/ultralytics/yolov5/tree/master), and if necessary `git pull` or `git clone` a new copy to ensure your problem has not already been solved in [master](https://github.com/ultralytics/yolov5/tree/master).
- ✅ **Unmodified** – Your problem must be reproducible using official YOLOv5 code without changes. [Ultralytics](https://ultralytics.com/) does not provide support for custom code ⚠️.

If you believe your problem meets all the above criteria, please close this issue and raise a new one using the 🐛 **Bug Report** [template](https://github.com/ultralytics/yolov5/issues/new/choose) with a [minimum reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) to help us better understand and diagnose your problem. 

Thank you! 😃
